Seasons Defense/Assets/Scripts/TowerDecider.cs

```diff
@@ -0,0 +1,32 @@
+using UnityEngine;
+
+public class TowerDecider : MonoBehaviour
+{
+    public GameObject leftTurret;
+    public GameObject middleTurret;
+    public GameObject rightTurret;
+
+    private float THIRD = Screen.width / 3f;
```

Comment:
My convention on Rider is telling me this should be called:
```suggestion
    private float _third= Screen.width / 3f;
```
If VS is telling you otherwise, then ignore this comment